f = 0;

Попытка
	Попытка
		f = f/0;
	Исключение
		f = f + 1;
		ВызватьИсключение "Вложенное исключение.";
		raise "Вложенное исключение.";
	КонецПопытки;

	f = f + 1;
	Сообщить(f);

Исключение
	Инфо = ErrorInfo();
    Сообщить("Описание='" + Инфо.Описание + "'");
    Сообщить("ИмяМодуля='" + Инфо.ИмяМодуля + "'");
    Сообщить("НомерСтроки=" + Инфо.НомерСтроки);
    Сообщить("ИсходнаяСтрока='" + Инфо.ИсходнаяСтрока + "'");
	Сообщить("Причина='" + Инфо.Причина + "'");
КонецПопытки; 